ui/src/views/network/InternalLBAssignVmForm.vue

```diff
@@ -210,17 +210,10 @@ export default {
         this.$pollJob({
           jobId: response.assigntoloadbalancerruleresponse.jobid,
           successMessage: `${this.$t('message.success.assigned.vms')} ${this.$t('label.to')} ${this.resource.name}`,
-          successMethod: () => {
```

Comment:
isFetchData: true ? So the page reloads after assigning a VM ?

Reply:
With condition `if (!action || !('isFetchData' in action) || (action.isFetchData))` in `utils/plugins.js`, even if isFetchData: true is not set, it will still be called after the polljob finish.